src/components/resources/index.js

```diff
@@ -129,13 +129,13 @@ const RESOURCES = [
         title: "Wallet SDK Swift",
         Svg: Graphics.walletSdkSwift,
         content: "Build Swift-based applications using our SDKs",
-        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-swift/documentation/atalaprismsdk/"
     },
     {
         title: "Wallet SDK TypeScript",
         Svg: Graphics.walletSdkTS,
         content: "Build web-based applications using our SDKs",
-        linkTo: "#"
+        linkTo: "https://input-output-hk.github.io/atala-prism-wallet-sdk-ts/"
     },
     {
         title: "Wallet SDK KMM",
```

Comment:
To `Edge Agent SDK KMP`
And `Build KMP-based`

Reply:
fixed